src/materials/large_deformation_models/HenckyIsotropicElasticity.C

```diff
@@ -95,3 +99,81 @@ HenckyIsotropicElasticity::computeMandelStressNoDecomposition(const ADRankTwoTen
 
   return stress;
 }
+
+ADRankTwoTensor
+HenckyIsotropicElasticity::computeMandelStressVolDevDecomposition(const ADRankTwoTensor & Fe,
+                                                                  const bool plasticity_update)
+{
+  ADRankTwoTensor strain = Fe;
+  // If this is called during a plasticity update, we need to first exponentiate Fe, where Fe should
+  // be some plastic flow. The foolowing operations cancel out with an exponentiation of Fe, so we
+  // only do this in the case of exponentiate == false
```

Comment:
```suggestion
  // only do this in the case of plasticity_update == false
```
I know you probably just copied it from some other place written by me. I think I said it wrong there as well.